### DISCUSSION TITLE
Roles to admin meeting guide

### DISCUSSION THREAD

**Discussion by DCO84770** [code4recovery/12-step-meeting-list#1100] [Q&A] [answered]
Looks like one must have master WP admin credentials to manage the Meeting Guide? I tried assigning someone as Editor and access to things like plugins are disabled. I want to have a backup person administer meetings but assigning master privileges just to do this is not ideal. Comments?

**Comment by kiyote33** [comment, marked as answer]
Hi @DCO84770, I've found the Editor role to be just perfect for administering the 12 Step Meeting List. Users in that role can do CRUD operations on the Meeting List but not a lot else. If that doesn't work for you, I believe there's a few plugins available for creating custom roles where you can tailor a role to suit your needs. You might try googling for one of them.
Hope this helps...

**Reply by DCO84770** [reply]
Thanks! We use CSV files. I'll look into role plug-ins. Thanks for the tip.

**Reply by joshreisner** [reply]
Would it help if we downgraded the restriction on the Import page to allow Editors? We used to have this combined with the Settings page, which makes sense to keep for admins only. But now that they're split I don't see why we couldn't open the Import page to editors, it's basically editing listings _en masse_.

**Reply by DCO84770** [reply]
Josh That would be great. Otherwise the only people who can upload meetings are admins, who can do almost anything. We have folks willing to assist in maintaining our meetings who (for now) are better suited to be editors.

**Reply by DCO84770** [reply]
Thanks Josh. I updated the plug-in and confirm Editor is able to access Mertinh settings. Great job!

**Comment by DCO84770** [comment]
We’ll done
